spec/models/auction_spec.rb

```diff
@@ -70,6 +70,16 @@
 
         expect(auction).to be_invalid
       end
+
+      it 'validates presence of c2_status' do
+        auction = create(:auction, published: :unpublished,
```

Comment:
let's just put all of these attributes on one line (I think they will fit?). If they are too long, I usually format with one attr on each line:

```
auction = create(
  :auction,
  published: :unpublished,
  purchase_card: :default
)
```

we also have an `:unpublished` trait in our factories so this can be simplified as:

`auction = create(:auction, :unpublished, purchase_card: :default`